packages/extension-browser/src/devtools/views/pages/results/header.css

```
@@ -32,3 +32,7 @@
 .showPassedHints {
     text-align: right;
 }
+
+.showPassedHints span {
```

Let's try to avoid descendant selectors here. I'd suggest updating `header.tsx` to set a new `showPassedHintsLabel` class name on the `LabelText`:

```tsx
<LabelText className={styles.showPassedHintsLabel}>
```

And then using that class name for this selector.

```suggestion
.showPassedHintsLabel {
```

---

@antross done!